Tests/kaas/k8s-version-policy/k8s-eol-data.yml

```diff
@@ -0,0 +1,12 @@
+# https://kubernetes.io/releases/patch-releases/#detailed-release-history-for-active-branches
+
+- branch: '1.29'
+  end-of-life: '2025-02-28'
+- branch: '1.28'
+  end-of-life: '2024-10-28'
+- branch: '1.27'
+  end-of-life: '2024-06-28'
+- branch: '1.26'
+  end-of-life: '2024-02-28'
+- branch: '1.25'
+  end-of-life: '2023-10-28'
```

Comment:
One thing that bothers me: this file needs to be maintained from now on. I couldn't find a machine readable version upstream, just the [Patch releases document](https://kubernetes.io/releases/patch-releases/#detailed-release-history-for-active-branches).

Reply:
We could assume that this rhythm of Oct 28, Feb 28, June 28 goes on, and then we only need to change anything if this rhythm breaks. Then of course we also need to know when these versions come out initially.

Reply:
Okay. With this scheme, another assumption that can break is that only the minor branch increments. I would like to take that idea into a separate issue.

For this PR, what do you think of issuing a warning if the file seems to be out of date at the time of script execution. This is the case if the file lists less than 3 non-EOL releases. And bail out if there are less than 2. With that approach we would at least see it if we monitor the logs in the CI.

Reply:
> issuing a warning if the file seems to be out of date at the time of script execution. This is the case if the file lists less than 3 non-EOL releases. And bail out if there are less than 2.

I've implemented this in the last commit. If you do not agree with that change, I can easily revert it, no problem :-)